src/protagonist/Orchestrator.Tests/Integration/PdfTestsNoLocalstack.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Threading;
+using DLCS.AWS.S3;
+using DLCS.AWS.S3.Models;
+using DLCS.Core.Types;
+using DLCS.Model.Assets;
+using DLCS.Model.Assets.NamedQueries;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.Extensions.DependencyInjection;
+using Orchestrator.Infrastructure.NamedQueries.Persistence;
+using Orchestrator.Tests.Integration.Infrastructure;
+using Test.Helpers.Integration;
+
+namespace Orchestrator.Tests.Integration;
+
+/// <summary>
+/// Tests pdf requests that cannot be tested using localstack
```

Comment:
Worth adding an explanation of why these can't use localstack + link to the issue? Hopefully once it's fixed we can roll these back into rest of tests.